pkg/interface/src/views/apps/launch/app.js

```diff
@@ -25,7 +25,7 @@ const ScrollbarLessBox = styled(Box)`
 
 export default function LaunchApp(props) {
   const [hashText, setHashText] = useState(props.baseHash);
-
+  console.log('it works')
```

Comment:
loose console.log

Reply:
sure thing, wasn't aware of the commit message convention